app/src/main/res/layout/feedinfo.xml

```diff
@@ -123,24 +87,103 @@
-                    tools:text="http://www.example.com/feed"
-                    tools:background="@android:color/holo_green_dark"/>
+                    tools:background="@android:color/holo_green_dark"
+                    tools:text="http://www.example.com/feed" />
 
             <TextView
-                    style="@style/AntennaPod.TextView.Heading"
-                    android:layout_width="match_parent"
-                    android:layout_height="wrap_content"
-                    android:layout_marginTop="4dp"
-                    android:layout_marginBottom="8dp"
-                    android:text="@string/description_label"/>
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:textSize="18sp"
+                android:layout_marginTop="16dp"
+                android:layout_marginBottom="4dp"
+                android:text="@string/description_label"
+                android:textColor="?android:attr/textColorPrimary"
+                tools:background="@android:color/holo_red_light" />
 
             <TextView
                     android:id="@+id/txtvDescription"
                     android:layout_width="match_parent"
                     android:layout_height="wrap_content"
-                    android:textIsSelectable="true"
                     android:text="@string/design_time_lorem_ipsum"
-                    tools:background="@android:color/holo_green_dark"/>
+                    android:textIsSelectable="true"
+                    tools:background="@android:color/holo_green_dark" />
+
+            <TextView
+                android:id="@+id/lblStatistics"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="16dp"
+                android:textSize="18sp"
+                android:layout_marginBottom="8dp"
+                android:text="@string/statistics_label"
+                android:textColor="?android:attr/textColorPrimary"
+                tools:background="@android:color/holo_red_light" />
+
+            <TableLayout
+                android:layout_width="match_parent"
+                android:layout_height="match_parent">
+
+                <TableRow
+                    android:layout_width="match_parent"
+                    android:layout_height="match_parent">
+
+                    <TextView
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"
+                        android:text="@string/statistics_listened_for" />
+
+                    <TextView
+                        android:id="@+id/txtvPodcastTime"
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:layout_marginLeft="5dp"
+                        android:text="TextView" />
```

Comment:
Please just remove the `android:text` attribute. With it, I can see the text `TextView` for a short moment before the actual data is shown. It would be better to just show nothing instead of showing something unrelated. Same for the other TextViews.